_data/versions.yml

```diff
@@ -252,6 +252,35 @@ v3.6:
       version: v3.6.0
 
 v3.5:
+- title: v3.5.8
+  components:
+    calico/node:
+      version: v3.5.8
+      url: https://github.com/projectcalico/node/releases/tag/v3.5.8
+    calicoctl:
+      version: v3.5.8
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.5.8
+    calico/cni:
+      version: v3.5.8
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.5.8
+    calico/kube-controllers:
+      version: v3.5.8
+      url: https://github.com/projectcalico/k8s-policy/releases/tag/v3.5.8
```

Comment:
Can you change this to be `kube-controllers` instead of `k8s-policy`? I guess at one point it was k8s-policy but I don't think it has been that for a long time, there is a redirect it seems to kube-controllers so it isn't wrong exactly but I think we should change it. I'm assuming it has just been getting copied from one to the next.